stable/mlrun-kit/templates/pipelines/configmaps/workflow-controller-configmap.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.pipelines.enabled -}}
+apiVersion: v1
+data:
+  artifactRepository: |
+    archiveLogs: true
+    s3:
+      endpoint: "{{ .Values.pipelines.minio.endpoint }}:{{ .Values.pipelines.minio.endpointPort }}"
+      bucket: "{{ .Values.pipelines.minio.bucket }}"
+      insecure: true
+      accessKeySecret:
+        name: mlpipeline-minio-artifact
+        key: accesskey
+      secretKeySecret:
+        name: mlpipeline-minio-artifact
+        key: secretkey
+  containerRuntimeExecutor: pns
+  executor: |
+    imagePullPolicy: IfNotPresent
+    resources:
+      requests:
+        cpu: 0.01
+        memory: 32Mi
+      limits:
+        cpu: 0.5
```

Comment:
low, isn't it?

Reply:
lest keep it for now  with this values if needed we will increase